docs/bigquery-usage.rst

```diff
@@ -512,7 +506,7 @@ First, create the job locally:
    >>> client = bigquery.Client()
    >>> source_table = dataset.table(name='person_ages')
    >>> destination_table = dataset.table(name='person_ages_copy')
-   >>> job = source_table.copy_to(destination_table)  # API request
+   >>> job = client.copy_table('copy-table', destination_table, source_table)
```

Comment:
- Why `Client.copy_table` instead of `Table.copy`? (Sorry if we've already discussed, it's out of my memory by now.)
- For the `job_name` could you use something with the word job in it so it's clear what it's doing in the signature?
- In the actual implementation the job name positional arg is just called `name`. `job_name` would be clearer. (Not sure if you want to fix in this PR or just file an issue.)